### PR TITLE
Release/5.4.X

### PR DESCRIPTION
**Pull request by QuentinNIO**

### CODE REVIEW

File: .Ressources/docs/MarkDown/Utils/G4d.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 
 ### templates
 
+### templates
```
Comment (tk463) [inline .Ressources/docs/MarkDown/Utils/G4d.md:4]: duplicate ?

File: README.md [hunk 2]
```diff
@@ -60,6 +60,7 @@ It might be similar to Oh-My-Zsh and it will fufill some of their features (in a
    * [gik](.Ressources/docs/MarkDown/Modules/gik.md)
    * [jic](.Ressources/docs/MarkDown/Modules/jic.md)
    * [jks](.Ressources/docs/MarkDown/Modules/jks.md)
+   * [tsu](.Ressources/docs/MarkDown/Modules/tsu.md)
```
Comment (tk463) [inline README.md:63]: tsu ?

Comment (QuentinNIO) [reply README.md:63]: damn, new module but should not be here

File: README.md [hunk 3]
```diff
@@ -71,6 +71,7 @@ It might be similar to Oh-My-Zsh and it will fufill some of their features (in a
    * [Documentation](.Ressources/docs/MarkDown/Utils/Documentation.md)
    * [G4d](.Ressources/docs/MarkDown/Utils/G4d.md)
      * [templates](.Ressources/docs/MarkDown/Utils/G4d-templates.md)
+     * [templates](.Ressources/docs/MarkDown/Utils/G4d-templates.md)
```
Comment (tk463) [inline README.md:74]: duplicate

**Pull request merged**